converter/solid_figures.py

```diff
@@ -15,15 +15,22 @@ class SolidFigure(ABC):
     position: tuple[float, float, float] = field(default_factory=lambda: [0., 0., 0.])
     rotation: tuple[float, float, float] = field(default_factory=lambda: [0., 0., 0.])
 
+    def expand(self, expansion: float) -> None:
+        """Expand figure by `expansion` in each dimension."""
```

Comment:
It is not clear how does it work. Provide an example.
For me if expansion is 5 and we have shpere with radius 2 and center (0,0,0) then it will have radius 10 after expansion. Is it so ?

Reply:
Maybe you mean simply `scaling_factor` here ?

Reply:
1. Yes, expansion by 1 on a 1by1by1 cube would result in a 2by2by2 cube.
2. No,  it expands each dimension by `expansion`. If we had a 2by1by1 cube when we expand it by 1 it becomes a 3by2by2 cube.

Reply:
So what kind of expansion it is ? Additive or multiplicative ?

Reply:
Its additive

Reply:
What about renaming `expansion` to `margin` ?
My first impression from reading the code was that its multiplicative